index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav id="phone">
+            <div class="logo">
+                <a href="#">My Logo</a>
+            </div>
+            <img src="images/humbarger.svg" alt="humbarger" class="humbarger">
+        </nav>
+    </header>
+    <main>
+
+        <section id="bg">
+            <section id="bio">
+                <h1 class="title">I’m Mariane Glad to see you!</h1>
```

Comment:
- [ ] [OPTIONAL] I think it would be great if you put your name here instead of `Mariane` to make the portfolio tailored to you. 😎️